bin/telemetry_sim_ATS_1VisitNight

```diff
@@ -125,20 +125,24 @@ if __name__ == "__main__":
         time.sleep(args.exptime)
 
         # Send startReadout Event after integration is finished
+        kw['timeStamp'] = time.time()
```

Comment:
I think we're supposed to use the time from SAL (which will be TAI), not from the system, but as I understand it this is just a simulation for testing so that's not a big problem.

Reply:
That's right. This is a stand-alone simulation.